hlua/src/rust_tables.rs

```diff
@@ -193,6 +191,45 @@ impl<'a, 'lua, L, T, E> PushOne<L> for &'a [T]
 {
 }
 
+impl<'lua, L> LuaRead<L> for HashMap<AnyHashableLuaValue, AnyLuaValue>
+    where L: AsMutLua<'lua>
+{
+    fn lua_read_at_position(lua: L, index: i32) -> Result<Self, L> {
```

Comment:
Please add a TODO saying that this should be implemented using the `LuaTable` API instead of raw Lua calls.
Right now the `LuaTable` API is very difficult to use and I'm not ever sure you wouldn't get unsolvable lifetime errors, so I only ask for a TODO.